store/tikv/2pc_test.go

```diff
@@ -42,7 +42,10 @@ func (s *testCommitterSuite) SetUpTest(c *C) {
 	mvccStore := mocktikv.NewMvccStore()
 	client := mocktikv.NewRPCClient(s.cluster, mvccStore)
 	pdCli := &codecPDClient{mocktikv.NewPDClient(s.cluster)}
-	store, err := newTikvStore("mock-tikv-store", pdCli, client, false)
+	spkv := &MockSafePointKV{
+		store: make(map[string]string),
+	}
```

Comment:
I think it would be better to add a `newMockSafepointKV` function.

Reply:
Ok, I have revised.